clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -3405,7 +3405,12 @@ public void consumerUnSubscribe(PubSubTopic topic, PartitionConsumptionState par
     Instant startTime = Instant.now();
```

Comment:
Nit:
Can we rename it to be `consumerUnSubscribeForStateTransition` to make the intention clear?

I hope other people won't use it accidentally.